src/rust/Dockerfile

```diff
@@ -120,66 +184,100 @@ USER nonroot
 # analyzer-dispatcher
 FROM rust-dist AS analyzer-dispatcher-deploy
 
+# Hadolint v2.8.0 appears to me mistaken, this is a false positive
+# https://github.com/hadolint/hadolint/wiki/DL3022
+# hadolint ignore=DL3022
```

Comment:
wtf this sucks lol

Reply:
also, possibly meant "appears to be mistaken"

Reply:
> wtf this sucks lol

Yeah, was gonna submit as issue for it.

> also, possibly meant "appears to be mistaken"

oops. fixed.

Reply:
https://github.com/hadolint/hadolint/issues/761